src/Tool.tsx

```diff
@@ -65,6 +66,7 @@ const Tool = () => {
                     links={getLocales(locales, locale, (selected) => {
                         if (selected !== locale) {
                             updateGlobals({locale: selected});
+                            addons.getChannel().emit('CHANGED_LOCALE', selected)
```

Comment:
This should be LOCALE_CHANGED

Reply:
Oh, my bad! Thank you for noticing! I'll fix it :)